h/static/scripts/group-forms/components/test/CreateEditGroupForm-test.js

```diff
@@ -300,7 +309,8 @@ describe('CreateEditGroupForm', () => {
       descriptionEl.simulate('input');
       setSelectedGroupType(wrapper, type);
 
-      await wrapper.find('form[data-testid="form"]').simulate('submit');
+      wrapper.find('form[data-testid="form"]').simulate('submit');
+      await delay(0);
```

Comment:
In this case, we do need to wait a tick for the save to complete. Make the fact that we're waiting for one tick explicit using `await delay(0)`.